src/page/dataverse/dataverse.tsx

```diff
@@ -6,17 +6,45 @@ import { pipe } from 'fp-ts/lib/function'
 import { useBreakpoint } from '@/hook/useBreakpoint'
 import { useAppStore } from '@/store/appStore'
 import { DataverseItemCard } from '@/component/card/dataverseItemCard/dataverseItemCard'
-import type { DataverseItemCardProps } from '@/component/card/dataverseItemCard/dataverseItemCard'
 import { Button } from '@/component/button/button'
 import Chip from '@/component/chip/chip'
 import { Icon } from '@/component/icon/icon'
 import type { IconName } from '@/component/icon/icon'
 import './dataverse.scss'
 
-type DataverseItem = 'dataspace' | 'dataset' | 'service'
 type FilterLabel = 'dataspaces' | 'datasets' | 'services' | 'all'
-type FilterValue = DataverseItem | 'all'
-export type DataverseItemDetails = DataverseItemCardProps
+type FilterValue = 'dataspace' | 'dataset' | 'service' | 'all'
+
+type DataverseItem = {
+  id: string
+  label: string
+  description: string
+}
+
+export type InternationalizedDescription = {
+  en: string
+  fr: string
+  de: string
+}
+
+type Governance = {
+  description: InternationalizedDescription
+}
+
+export type Service = DataverseItem & {
+  type: 'service'
+}
+
+export type Dataset = DataverseItem & {
+  type: 'dataset'
+}
+
+export type DataSpace = DataverseItem & {
+  type: 'dataspace'
+  governance: Governance
+}
```

Comment:
You could have used a `Typed` type to enforce the typing of the `type` property (not sure to be clear), but given that the model is going to change soon, it may not be worth the effort right now. It's up to you.

```suggestion
type Typed<T extends 'service' | 'dataset' | 'dataspace'> = { type: T }

export type Service = DataverseItem & Typed<'service'>

export type Dataset = DataverseItem & Typed<'dataset'>

export type DataSpace = DataverseItem &
  Typed<'dataspace'> & {
    governance: Governance
  }
```